spec/workspace-element-spec.coffee

```diff
@@ -47,9 +47,9 @@ describe "WorkspaceElement", ->
 
     it "updates the font-family based on the 'editor.fontFamily' config value", ->
       initialCharWidth = editor.getDefaultCharWidth()
-      expect(getComputedStyle(editorElement).fontFamily).toBe atom.config.get('editor.fontFamily')
+      expect(getComputedStyle(editorElement).fontFamily).toBe atom.config.get('editor.fontFamily') + ", 'Apple Color Emoji'"
```

Comment:
Right, but I don't think it's worth adding an extra conditional in the test considering we don't run them on other platforms. Given how easy this change was, I believe it's okay to keep it as-is for now, and revisit it later when adding compatibility for other platforms. :+1: